Dev 877 update readme md to match new configurations (#652)
bepronetwork/web-network: docs/GITHUB-GATEWAY.md

## 1. GITHUB REPOSITORYS MANAGER

In order to manage repositories issues and pull requests, the application requires a Github Token. This should be treated as a bot account.

Follow this [documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) to generator a Github Token, remember of guarantee the permissions of scopes `repo` and `admin:repo_hook` and change `Expiration` to a highest time.

Update the .env file with the token and user;

```text
# .env

# Create new github token for Bot
NEXT_GH_TOKEN=yourGithubToken
NEXT_PUBLIC_GH_USER=yourGithubUserLogin
```

## 2. DEFAULT REPOSITORY/BRANCH

> It's necessary configure the default branch and repository, for ocasion no have others repositories registred these. `master` is a default branch.

Reviewer: ```suggestion
It's necessary to configure the default branch and repository, for ocassion of not have others repositories this will be registered. `master` is the default branch.
```

> This repository will be registered and available on only default network.

Reviewer: ```suggestion
This repository will be registered and available only for the default network.
```
